### PR TITLE
Add Error pages

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
Fix: #168247836

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
screencapture-localhost-3000-notfound-2019-09-10-12_44_16
screencapture-localhost-3000-server-2019-09-10-12_43_00

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/pages/NotFound.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Page from '../components/Page';
+
+import config from '../config';
+import { TitlePageHeader } from '../components/Header';
+import ErrorContent from '../components/ErrorContent';
+
+function ServerError() {
```
Comment (kilemensi) [inline src/pages/NotFound.js]: ~`ServerError`~`NotFound`

File: src/App.js [hunk 2]
```diff
@@ -14,13 +16,15 @@ function App() {
...
         <Route exact path="/about" component={About} />
+        <Route exact path="/500" component={ServerError} />
         <Route exact path="/:country" component={Country} />
         <Route exact path="/profile/:geoId" component={Profile} />
         <Route
           exact
           path="/compare/:geoId/vs/:comparisonGeoId"
           component={Profile}
         />
+        <Route path="*" component={NotFound} />
```
Comment (kilemensi) [inline src/App.js]: No need for `path="*"`

File: src/components/ErrorContent.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+function ErrorContent({ classes, title, description }) {
+  return (
+    <Grid
+      container
+      direction="row"
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item xs={12} sm={12} md={8} lg={8} xl={8}>
```
Comment (kilemensi) [inline src/components/ErrorContent.js]: `<Grid item xs={12} md={8}>` should do it... higher sizes takes whatever the closes lower size was set.

File: src/components/About/AboutCountry.js [hunk 4]
```diff
@@ -47,10 +47,7 @@ const styles = theme => ({
 
 function AboutCountry({ classes, dominion }) {
   const { selectedCountry = {} } = dominion;
-  const info = config.about[selectedCountry.slug] || {
-    intro: 'Not found',
-    other: 'Not found'
-  };
+  const info = config.about[selectedCountry.slug] || {};
```
Comment (kilemensi) [inline src/components/About/AboutCountry.js]: If we're sure `info` will always be there, lets remove the ` || {};` as well.

File: src/pages/ServerError.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import ErrorContent from '../components/ErrorContent';
+
+function ServerError() {
+  return (
+    <Page>
+      <TitlePageHeader dominion={config} profile={{}} />
+      <ErrorContent
+        title="500 - Internal Server Error"
+        description="We are having some trouble processing your request.
+              We have logged the error and will investigate. You can try again or if the issues persisit,
```
Comment (kilemensi) [inline src/pages/ServerError.js]: 1. `You can try again *later* and if...` or `Please try again *later* and if ...`,
2. Link `contact us` to contact page.

File: src/components/ErrorContent.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        <Grid className={classes.title}>
```
Comment (kilemensi) [inline src/components/ErrorContent.js]: We're not using any of `<Grid` features here so normal `div` should do it.

**Comment by nerdlet** [comment]
@kilemensi Done

File: src/App.js [hunk 7]
```diff
@@ -15,13 +17,15 @@ function App() {
...
-        <Route exact path="/:country" component={Country} />
+        <Route exact path="/500" component={ServerError} />
+        <Route exact path="/:country(kenya|southfrica)" component={Country} />
         <Route exact path="/profile/:geoId" component={Profile} />
         <Route
           exact
           path="/compare/:geoId/vs/:comparisonGeoId"
           component={Profile}
         />
+        <Route component={NotFound} />
```
Comment (kilemensi) [inline src/App.js]: The `NotFound` route *must* be the _last_ route.

File: src/components/About/AboutCountry.js (hunk 4 above)
Comment (kilemensi) [reply src/components/About/AboutCountry.js]: I meant if we're sure `config.about[selectedCountry.slug]` will always contain `info`, then there is no need to add ` || {}` to it.

**Pull request merged**